mypyc/lib-rt/CPy.h

```diff
@@ -64,6 +64,14 @@ typedef struct tuple_T4CIOO {
 } tuple_T4CIOO;
 #endif
 
+// System-wide empty tuple constant
+extern PyObject * __mypyc_empty_tuple__;
+
+static inline PyObject *_CPyTuple_LoadEmptyTupleConstant() {
+    // do tests still pass if I comment this out? empty tuple singleton is not tracked by gc
+    // Py_INCREF(__mypyc_empty_tuple__);
```

Comment:
I believe I can take out this Py_INCREF since the empty tuple singleton is not tracked by the GC but I want to wait for confirmation before I get rid of the comment